### PR TITLE
Add "Preview Components" to Storybook, clean up "vNext" term

### PR DESCRIPTION
**Pull request by levithomason**
## Current Behavior

We mix "alpha" or "preview" components in with our officially released components. Example, Card is marked ALPHA and not ready for use but Button is not. Both appear in the same component list.

This PR also replaces "Fluent UI React _**vNext**_" with either "Fluent UI React Components" or the actual version number, where appropriate.

## New Behavior

- Added a "Preview Components" section to Storybook and includes: Input, Slider, and Card.
- Replaced "vNext" with appropriate version or library name.
- Standardized our preview/alpha component warning.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |         Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                | `48.488 kB`<br />`14.347 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          | `43.997 kB`<br />`13.195 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |   `7.618 kB`<br />`3.232 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |   `8.885 kB`<br />`3.673 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |   `7.811 kB`<br />`3.361 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `165.87 kB`<br />`46.955 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            | `32.489 kB`<br />`10.636 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |  `21.474 kB`<br />`7.118 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |  `22.936 kB`<br />`7.751 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/b154c0cbbb4f87fb7480b6313e212d6e7ff0bd01'>b154c0cbbb4f87fb7480b6313e212d6e7ff0bd01</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">179</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">214</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Avatar-mount_base.html">766</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Avatar-mount.html">783</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/BaseButton-mount_base.html">811</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/BaseButton-mount.html">822</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Breadcrumb-mount_base.html">2287</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Breadcrumb-mount.html">2302</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ButtonNext-mount_base.html">443</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ButtonNext-mount.html">444</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Checkbox-mount_base.html">1346</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Checkbox-mount.html">1317</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/CheckboxBase-mount_base.html">1149</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/CheckboxBase-mount.html">1121</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ChoiceGroup-mount_base.html">4132</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ChoiceGroup-mount.html">4112</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ComboBox-mount_base.html">863</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ComboBox-mount.html">861</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/CommandBar-mount_base.html">8893</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/CommandBar-mount.html">8958</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ContextualMenu-mount_base.html">7330</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ContextualMenu-mount.html">7313</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/DefaultButton-mount_base.html">1005</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/DefaultButton-mount.html">1009</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/DetailsRow-mount_base.html">3262</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/DetailsRow-mount.html">3211</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/DetailsRowFast-mount_base.html">3236</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/DetailsRowFast-mount.html">3189</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/DetailsRowNoStyles-mount_base.html">3070</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/DetailsRowNoStyles-mount.html">3082</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Dialog-mount_base.html">2227</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Dialog-mount.html">2258</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/DocumentCardTitle-mount_base.html">157</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/DocumentCardTitle-mount.html">157</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Dropdown-mount_base.html">2768</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Dropdown-mount.html">2791</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FluentProviderNext-mount_base.html">1692</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FluentProviderNext-mount.html">1694</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FluentProviderWithTheme-mount_base.html">114</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FluentProviderWithTheme-mount.html">120</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">97</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FluentProviderWithTheme-virtual-rerender.html">106</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">179</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">214</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FocusTrapZone-mount_base.html">1600</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FocusTrapZone-mount.html">1548</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FocusZone-mount_base.html">1591</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/FocusZone-mount.html">1581</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/IconButton-mount_base.html">1500</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/IconButton-mount.html">1510</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Label-mount_base.html">318</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Label-mount.html">317</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Layer-mount_base.html">2576</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Layer-mount.html">2507</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Link-mount_base.html">436</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Link-mount.html">438</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/MakeStyles-mount_base.html">1475</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/MakeStyles-mount.html">1487</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/MenuButton-mount_base.html">1283</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/MenuButton-mount.html">1285</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/MessageBar-mount_base.html">1716</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/MessageBar-mount.html">1730</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Nav-mount_base.html">2833</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Nav-mount.html">2879</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/OverflowSet-mount_base.html">969</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/OverflowSet-mount.html">977</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Panel-mount_base.html">2136</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Panel-mount.html">2146</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Persona-mount_base.html">753</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Persona-mount.html">734</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Pivot-mount_base.html">1267</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Pivot-mount.html">1248</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/PrimaryButton-mount_base.html">1135</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/PrimaryButton-mount.html">1131</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Rating-mount_base.html">6654</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Rating-mount.html">6673</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/SearchBox-mount_base.html">1152</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/SearchBox-mount.html">1152</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Shimmer-mount_base.html">2143</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Shimmer-mount.html">2199</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Slider-mount_base.html">1670</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Slider-mount.html">1702</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/SpinButton-mount_base.html">4324</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/SpinButton-mount.html">4343</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Spinner-mount_base.html">393</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Spinner-mount.html">396</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/SplitButton-mount_base.html">2739</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/SplitButton-mount.html">2711</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Stack-mount_base.html">470</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Stack-mount.html">457</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/StackWithIntrinsicChildren-mount_base.html">1985</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/StackWithIntrinsicChildren-mount.html">1971</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/StackWithTextChildren-mount_base.html">4491</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/StackWithTextChildren-mount.html">4518</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/SwatchColorPicker-mount_base.html">9875</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/SwatchColorPicker-mount.html">9733</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/TagPicker-mount_base.html">2406</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/TagPicker-mount.html">2275</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/TeachingBubble-mount_base.html">11336</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/TeachingBubble-mount.html">11348</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Text-mount_base.html">384</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Text-mount.html">386</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/TextField-mount_base.html">1230</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/TextField-mount.html">1199</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ThemeProvider-mount_base.html">1035</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ThemeProvider-mount.html">1059</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ThemeProvider-virtual-rerender_base.html">551</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ThemeProvider-virtual-rerender.html">540</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1637</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1609</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Toggle-mount_base.html">714</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/Toggle-mount.html">717</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/buttonNative-mount_base.html">128</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21541/perf-test/buttonNative-mount.html">136</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9423"/>

File: packages/react-card/src/Card.stories.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ export { ActionCard } from './stories/CardAction.stories';
 export { GridCard } from './stories/CardGrid.stories';
 
 export default {
-  title: 'Components/Card [ALPHA]',
+  title: 'Preview Components/Card',
```
Comment (levithomason) [inline packages/react-card/src/Card.stories.tsx:9]: Listed at the bottom of the left nav:

image

File: packages/react-card/src/CardDescription.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
-**This component is available for limited production use, please contact us if you intend to use it in your product.**
-
-**The APIs will change before final release as functionality is still being added.**
+> **⚠️ Preview components are considered unstable:**
+>
+> - Features and APIs may change before final release
+> - Please contact us if you intend to use this in your product
```
Comment (levithomason) [inline packages/react-card/src/CardDescription.md:4]: I combined the various warning messages into a single message and made sure all 3 preview components include it.


image

File: packages/react-components/.storybook/theme.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ const theme = create({
   inputBorderRadius: 4,
 
   // Use the fluent branding for the upper left image
-  brandTitle: 'Fluent UI React vNext Components',
+  brandTitle: 'Fluent UI React Components',
```
Comment (levithomason) [inline packages/react-components/.storybook/theme.js:36]: Bye bye "vNext"

File: packages/react-components/public/intro.css [hunk 4]
```diff
@@ -58,6 +58,13 @@ h1.fluent {
   letter-spacing: -0.16px;
 }
 
+h1 .fluent-version {
+  display: block;
+  font-size: 24px; /* --font-size-base-600 */
+  line-height: 32px;
+  color: #707070; /* --color-neutral-foreground-3 */
+}
```
Comment (levithomason) [inline packages/react-components/public/intro.css:66]: Added the actual version number to the intro heading in lieu of vNext:

image

**Comment by micahgodbolt** [comment]
this will cause a bit of churn with my on my PR, but nothing huge. I would like to get my PR in soon though to create less chance of churn

**Review by micahgodbolt — APPROVED** [review]

**Pull request merged**